presto-native-execution/presto_cpp/main/connectors/arrow_flight/tests/utils/Utils.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <arrow/api.h>
+
+#include "presto_cpp/main/connectors/arrow_flight/Macros.h"
+#include "velox/common/base/Exceptions.h"
+
+namespace facebook::velox {
+class StringView;
+} // namespace facebook::velox
+
+namespace facebook::presto::test {
+
+template <typename T>
+std::shared_ptr<arrow::Array> makeNumericArray(
```

Comment:
Nit : Add an alias for std::shared_ptr<arrow::Array> to reuse 

using ArrowArrayPtr = std::shared_ptr<const arrow::Array>;